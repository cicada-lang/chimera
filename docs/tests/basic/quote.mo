print quote f(x)
print quote a.f(x)
print quote a.b.c.f(x)
