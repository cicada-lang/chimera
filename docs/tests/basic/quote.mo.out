f(x)
a.f(x)
a.b.c.f(x)
